app/mbain_whisperx/core.py

```diff
@@ -0,0 +1,99 @@
+import os
+from typing import BinaryIO, Union
+from io import StringIO
+from threading import Lock
+import torch
+import whisper
+import whisperx
+from whisper.utils import ResultWriter, WriteTXT, WriteSRT, WriteVTT, WriteTSV, WriteJSON
+
+model_name= os.getenv("ASR_MODEL", "base")
+hf_token= os.getenv("HF_TOKEN", "")
+x_models = dict()
+
+if torch.cuda.is_available():
+    device = "cuda"
+    model = whisper.load_model(model_name).cuda()
```

Comment:
Why not use whisperx model for transcription?

Reply:
Thank you for your feedback! I fixed this issue, so whisperx should also be used for transcription now.